tests/core_tests/chaingen.h

```diff
@@ -549,7 +549,7 @@ inline bool do_replay_file(const std::string& filename)
         std::vector<crypto::public_key> spend_public_keys; \
         for (const auto &k: all_multisig_keys) \
           spend_public_keys.push_back(k); \
-        crypto::public_key spend_pkey = cryptonote::generate_multisig_N1_N_spend_public_key(spend_public_keys); \
+        crypto::public_key spend_pkey = cryptonote::generate_multisig_M_N_spend_public_key(spend_public_keys); \
```

Comment:
Can you add some tests for the new threshold cases ?

Reply:
I can... But the problem is core tests don't use wallet or it's parts and I need to copy and paste some code into appropriate macro...